### PR TITLE
SB-56: Fix Typescript build errors

### PR DESCRIPTION
**Pull request by EzeLamar**
Changes:
* Fix Typescript error detected during the build process of the project.
* Update README with the steps to possibilitate the building process.  
* Add Error Helper for the catching process of the Promises.
* Add script to execute lint and prettier fixes with only one command.

### CODE REVIEW

**Comment by EzeLamar** [comment]
[Deploy App to Firebase Hosting](https://trello.com/c/DfCCN316/56-deploy-app-to-firebase-hosting)

File: app/utils/Error/ErrorHelper.ts
Comment (lubualo) [inline app/utils/Error/ErrorHelper.ts:1]: This was mindblowing.

File: README.md [hunk 1]
```diff
@@ -56,3 +56,24 @@ To start the storybook local server execute the next command:
...
+```shell
+npm run lint:check
+npm run format:check
+```
+
+Then, verify there is no Typescript errors with the next command:
+
+```shell
+npx tsc --noEmit
+```
```
Comment (lubualo) [inline README.md:73]: I was wondering if you could add this to the Husky pre-commit routines? Unless you think of a reason for not doing it, please add it. There is no need to request a new CR if you add just that line to Husky.

**Review by lubualo — APPROVED** [review]
It seems fine to me. Please, read my comment before merging.

File: README.md (hunk 1 above)
Comment (EzeLamar) [reply README.md:73]: Sounds good to me, but in a near future we will have to review the lint and prettier checks to make them consistent.

**Pull request merged**